frontend/src/components/attendances/AttendanceCard.js

```diff
@@ -73,22 +75,80 @@ const AttendanceCardWrapper = styled(Paper)`
   }
 `
 
+//TODO: localize
+const labelEnrolledDate = 'Enrolled Date　:'
+const labelGrade = 'Grade　:'
+const labelYear = 'Year　:'
+const labelSchool = 'School　:'
+
 const AttendanceCard = ({ attendance, onEdit }) => {
   const deactivatedClass = classnames({
     deactivated: Boolean(attendance.student.deactivated_at),
   })
 
   return (
     <AttendanceCardWrapper className={deactivatedClass}>
-      <div className="row">
-        <ThingahaName className="thingaha-name w25">
-          {attendance.student.name}
-        </ThingahaName>
-        <div className="iconTextWrapper w25">
-          <h5 className="capName">Donator Name : </h5>
-          <div className="smallText">{attendance.grade}</div>
-        </div>
-        <div className="w50 text-right">
+      <Grid container spacing={1}>
+        <Grid item sm={12} xs={12}>
+          <Grid container spacing={1}>
+            <Grid item xs={12}>
+              <ThingahaName className="thingaha-name">
+                {attendance.student.name}
+              </ThingahaName>
+            </Grid>
+            <Box
+              display={{ xs: 'block', sm: 'block' }}
+              style={{ marginLeft: '-4rem' }}
+            >
+              <Link
+                to={`/attendances/${attendance.id}`}
+                params={{ attendancename: attendance.student.name }}
+              >
+                <VisibilityRoundedIcon
+                  color="primary"
+                  className="show"
+                  variant="rounded"
+                />
+              </Link>
+              <EditIcon
+                color="primary"
+                className="edit"
+                variant="rounded"
+                onClick={() => {
+                  onEdit(attendance)
+                }}
+              />
+            </Box>
+          </Grid>
+        </Grid>
+        <Grid item sm={6} xs={12}>
+          <div className="iconTextWrapper">
+            <h5 className="capName">{labelEnrolledDate} </h5>
+            <div className="smallText">{attendance.enrolled_date}</div>
+          </div>
+        </Grid>
+        <Grid item sm={6} xs={6}>
+          <div className="iconTextWrapper">
+            <h5 className="capName">{labelGrade} </h5>
+            <div className="smallText">{attendance.grade}</div>
+          </div>
+        </Grid>
+        <Grid item sm={6} xs={6}>
+          <div className="iconTextWrapper">
+            <h5 className="capName">{labelYear} </h5>
+            <div className="smallText">({attendance.year})</div>
+          </div>
+        </Grid>
+        <Grid item sm={6} xs={12}>
+          <div className="iconTextWrapper">
+            <h5 className="capName">{labelSchool} </h5>
+            <div className="smallText">{attendance.school.name}</div>
+          </div>
+        </Grid>
+        {/* <Box
```

Comment:
If this is no longer needed, can you remove it please?

Reply:
its my careless mistake 😄 ,
fixed